airflow/www/static/js/cluster-activity/live-metrics/Health.tsx

```diff
@@ -0,0 +1,99 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from "react";
+import {
+  Badge,
+  BoxProps,
+  Card,
+  CardBody,
+  CardHeader,
+  Center,
+  Flex,
+  Heading,
+  Stack,
+  Text,
+} from "@chakra-ui/react";
+import { useHealth } from "src/api";
+import type { API } from "src/types";
+import Time from "src/components/Time";
+import LoadingWrapper from "src/components/LoadingWrapper";
+
+const StatusRow = ({ status }: { status?: API.HealthStatus }) => (
+  <Stack direction="row">
+    <Text textIndent="15px">status:</Text>
+    <div>
+      <Badge
+        colorScheme={status === "healthy" ? "green" : "red"}
+        fontSize="0.9rem"
+      >
+        {status || "unknown"}
```

Comment:
If it's unknown, should it still be red?

Reply:
I'm not sure this even has a real use case now that I look at it close. I think `get_airflow_health` cannot return null or underfined status, it's either `healthy` or `unhealthy`. This is just to comply with typescript and handle those cases but might never get displayed. (also I feel like anything that is not 'green' is bad and should be displayed in red, cannot think of a use case where you would want your scheduler or medatabase not in a healthy state)